### PR TITLE
chore: change css files import to fit with Next 13

### PR DESCRIPTION
**Pull request by JoeyDjr**
This PR changes the way the CSS files are imported due to an error during the build of the project revealed in this PR https://github.com/okp4/portal-web/pull/105

:bug: The error and its fix are explained here : https://nextjs.org/docs/messages/css-global

_An attempt to import Global CSS from a file other than [pages/_app.js](https://nextjs.org/docs/advanced-features/custom-app) was made.
Global CSS cannot be used in files other than your [Custom <App>](https://nextjs.org/docs/advanced-features/custom-app) due to its side-effects and ordering problems._

:adhesive_bandage: There are two ways to fix this error : 

1. The first one (the one chosen) is to import all CSS files into `styles.scss` instead of doing it in each component file. 
_To avoid conflicts, relocate all first-party Global CSS imports to your [pages/_app.js file](https://nextjs.org/docs/advanced-features/custom-app)._
2. The second one (the one preferred but not chosen) is to [update our component to use local CSS (Component-Level CSS) via CSS Modules](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css).

:monocle_face: Why didn't I choose the second solution?
Because I felt it was overkilled : 

1. Rename CSS files :  `datasetInformation.scss` to `DatasetInformation.module.scss`
2. The import of the CSS file : `import './datasetInformation.scss'` to `import styles from './DatasetInformation.module.scss'`
3. Use the styles by doing something like : 
```tsx
<div className={styles.okp4_dataset_information_container}>
```
instead of 
```tsx
<div className="okp4-dataset-information-container">
```

:arrow_up: And then this PR upgrades the Next.js version to 13.0.1

🙏 Don't hesitate to tell me if anything is wrong @fredericvilcot @ccamel

### CODE REVIEW

**Comment by fredericvilcot** [comment]
Thanks @JoeyDjr for your investigation.
It looks good to me, that's pretty much better than using CSS modules, IMHO.

File: src/pages/dataverse/explore/dataspace/[dataspaceId]/dataset/[datasetId].tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
   PageTitle,
   PreviousPageButton
 } from '../../../../../../components'
-import './datasetId.scss'
+// import './datasetId.scss'
```
Comment (fredericvilcot) [inline src/pages/dataverse/explore/dataspace/[dataspaceId]/dataset/[datasetId].tsx:16]: Please remove commented imports

File: src/pages/dataverse/explore/index.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { useCallback, useEffect, useMemo, useState } from 'react'
 import type { GetServerSideProps, GetServerSidePropsResult, NextPage } from 'next'
 import type { DeepReadonly, SelectOption, SelectValue, UseState } from '@okp4/ui'
-import './explore.scss'
+// import './explore.scss'
```
Comment (fredericvilcot) [inline src/pages/dataverse/explore/index.tsx:4]: Please remove commented imports

**Review by fredericvilcot — CHANGES_REQUESTED** [review]
Great, just some format remarks

**Review by fredericvilcot — APPROVED** [review]
Great!

**Review by ccamel — COMMENTED** [review]
Do you really need to change the version of `Next.js` here? It would be better to let dependabot do it through the PR #96.

**Comment by JoeyDjr** [comment]
> Do you really need to change the version of `Next.js` here? It would be better to let dependabot do it through the PR #96.

@ccamel I cherry picked a dependabot commit that upgrade the version to `"13.0.1"`. So I think we can close https://github.com/okp4/portal-web/pull/96 and merge this PR to have the most recent version with the fixes.
Are you ok with that ?

**Review by ccamel — APPROVED** [review]
That's good. Thx :+1:

**Pull request merged**